core/src/test/java/org/pentaho/platform/workitem/WorkItemLifecyclePhaseTest.java

```diff
@@ -17,42 +17,64 @@
+    assertEquals( "Dispatched ", WorkItemLifecyclePhase.DISPATCHED.getPaddedName() );
+    assertEquals( "Received   ", WorkItemLifecyclePhase.RECEIVED.getPaddedName() );
+    assertEquals( "Rejected   ", WorkItemLifecyclePhase.REJECTED.getPaddedName() );
+    assertEquals( "In progress", WorkItemLifecyclePhase.IN_PROGRESS.getPaddedName() );
+    assertEquals( "Succeeded  ", WorkItemLifecyclePhase.SUCCEEDED.getPaddedName() );
+    assertEquals( "Failed     ", WorkItemLifecyclePhase.FAILED.getPaddedName() );
+    assertEquals( "Restarted  ", WorkItemLifecyclePhase.RESTARTED.getPaddedName() );
+    assertEquals( "Redirected ", WorkItemLifecyclePhase.REDIRECTED.getPaddedName() );
   }
 
   @Test
   public void testLifecycleDescriptions() {
-    Assert.assertEquals( "The work item has been submitted to the component responsible for its execution",
+    assertEquals( "The work item has been submitted to the component responsible for its execution",
       WorkItemLifecyclePhase.SUBMITTED.getDescription() );
-    Assert
-      .assertEquals( "The work item has been dispatched to the component responsible for its execution",
-        WorkItemLifecyclePhase.DISPATCHED.getDescription() );
-    Assert.assertEquals( "The work item has been received by the component responsible for its execution",
+    assertEquals( "The work item has been dispatched to the component responsible for its execution",
+      WorkItemLifecyclePhase.DISPATCHED.getDescription() );
+    assertEquals( "The work item has been received by the component responsible for its execution",
       WorkItemLifecyclePhase.RECEIVED.getDescription() );
-    Assert.assertEquals( "The work item execution has been rejected",
+    assertEquals( "The work item execution has been rejected",
       WorkItemLifecyclePhase.REJECTED.getDescription() );
-    Assert
-      .assertEquals( "The work item execution is in progress",
-        WorkItemLifecyclePhase.IN_PROGRESS.getDescription() );
-    Assert.assertEquals( "The work item execution has succeeded",
+    assertEquals( "The work item execution is in progress",
+      WorkItemLifecyclePhase.IN_PROGRESS.getDescription() );
+    assertEquals( "The work item execution has succeeded",
       WorkItemLifecyclePhase.SUCCEEDED.getDescription() );
-    Assert.assertEquals( "The work item execution has failed",
+    assertEquals( "The work item execution has failed",
       WorkItemLifecyclePhase.FAILED.getDescription() );
-    Assert.assertEquals( "The work item execution has been restarted",
+    assertEquals( "The work item execution has been restarted",
       WorkItemLifecyclePhase.RESTARTED.getDescription() );
+    assertEquals( "The work item execution has been redirected",
+      WorkItemLifecyclePhase.REDIRECTED.getDescription() );
+  }
+
+  @Test
+  public void testGetMaxNameLength() {
+    // our longest name is that of IN_PROGRESS
+    assertEquals( WorkItemLifecyclePhase.IN_PROGRESS.getName().length(), WorkItemLifecyclePhase.getMaxNameLength() );
```

Comment:
This doesn't feel right. The existing test case should not break when we just added another state